recipes/publish.rb

```diff
@@ -50,45 +50,17 @@
   #########################################################################
 
   # update a data bag with the artifact build info
-  # TODO: (jtimberman) This is not the first time this has been used in
-  # a delivery build cookbook. It's probably time to create a Chef
-  # resource for this functionality
-  ruby_block 'track-artifact-data' do # ~FC014
-    block do
-      load_delivery_chef_config
-      proj = Chef::DataBag.new
-      proj.name(project_name)
-      proj.save
+  load_delivery_chef_config
+  chef_data_bag project_name
```

Comment:
Is Cheffish guaranteed to be available? How does it ship? Core chef? A cookbook?